### ISSUE TITLE
Blender could not import the generated RDC file

### ISSUE THREAD

**Issue by xTheJim** [eliemichel/MapsModelsImporter#170]
Getting an error on import into Blender. The test files work fine on import.

Using:

- Blender 2.92
- Renderdoc 1.1

![image](https://user-images.githubusercontent.com/83611484/116943045-a4971980-ac27-11eb-8caf-9d53d07e691b.png)

**Comment by jarnie123** [comment]
I am getting this error too. Yet version 28.3 of Blender doesn't get the error.

**Comment by hung3108** [comment]
I using model map 0.3.7, render 1.10 and Blender 2.9.2 but I don't import on bleder

**Comment by jarnie123** [comment]
Please advise what is ' Model Map 0.3.7'? I Googled Model Map and got varying things. Is it a stand alone application or an addon? Where can it be downloaded?

**Comment by hung3108** [comment]
https://github.com/eliemichel/MapsModelsImporter/releases
you can download

**Comment by jarnie123** [comment]
Thanks. 

That is what I am using although I have MapsModelsImporter 0.3.1 and the web site now shows version 0.3.7 which is a later version although I only downloaded what I believed was the latest version a week ago so maybe there was a problem with 0.3.1 which was fixed in 0.3.7

Also the web site states to use Renderdoc 1.10 not 1.11 and I am using 1.9 which is the latest version. I don't know if that also makes a difference but I will try changing the MapsModelsImporter addon to version 0.3.7 first and see if that works.

**Comment by hung3108** [comment]
I created file  .rdc form render but when I import on the blender, my blender don't show terrain

**Comment by jarnie123** [comment]
Yes  You need to select the 'viewpoint shading' icon which is near the top right of screen.
![blender set texture](https://user-images.githubusercontent.com/71200790/117522732-5e133780-aff8-11eb-9afc-de611eee8ae1.jpg)

**Comment by hung3108** [comment]
when I import file rdc my blender didn't show image

![image](https://user-images.githubusercontent.com/83716565/117534678-b78d5d80-b01c-11eb-9487-465e80a1cbe0.png)

**Comment by hung3108** [comment]
![Uploading image.png…]()

**Comment by jarnie123** [comment]
RenderDoc  ver 1.9
Blender ver 2.92
ModelMapImporter ver 0.3.7

**Comment by hung3108** [comment]
Can you give me gmail address I will send rdc for you. Then can you import
to your blender?

**Comment by jarnie123** [comment]
No BUT you can download an rdc from me and see if you can import.
I can import no problems so if you can't import is something wrong with your Blender and addon.
If you can import then you are not creating an rdc using RenderDoc in the form required.
[https://mega.nz/file/S4YgDbzS#XxTAWUOS4HnUCcK7GjDxXyk86we57-6i09lq6IAocE4]

**Comment by hung3108** [comment]
Are you capture render on Google map or Google earth?

**Comment by jarnie123** [comment]
Please watch the video tutorial I mentioned earlier - I did it exactly as the video shows.

**Comment by jarnie123** [comment]
You can download my notes on how to get a 3d map using RenderDoc here -
[https://mega.nz/file/D8YACZLQ#lvffCxssX1CPBINPBfaCPpcnKdGtCIU42ZhTxrxabk0](url)

**Comment by hung3108** [comment]
I think Model map importer only support some area as well as countries.

**Comment by jarnie123** [comment]
The ONLY problem as I see it is that you need an area of the world which has 3D buildings in Google Maps. If you only see a flat image in Google Maps then you are wasting your time. The major cities in most modern countries appear to have 3D imagery however smaller places (such as Sechelt in Canada) does not. Vancouver in Canada does.
What area area you interested in (name of town or city and country).

**Comment by hung3108** [comment]
Yeah, I think google map and earth not support 3d map for some countries
such as Viet nam so I can't build terrain.

**Comment by SenecaDriver** [comment]
I can reproduce this with any version of RenderDoc after 1.10.  Once I downgrade to RenderDoc 1.10 MapsModelImporter works just fine.

**Comment by megha580** [comment]
hello , i tried to make a 3d view of sydney but still shows error in mapsmodelsimporter after importing the rdc file in Blender

**Comment by eliemichel** [comment]
Did you have a look at the suggestions from the Troubleshooting section? https://github.com/eliemichel/MapsModelsImporter#troubleshooting

**Comment by nimitz2000** [comment]
Possible solution worked for me : 
INSTALL redistributables (Last) for Visual C++ (2015-2019) and remove import maps addon from blender, later re install add on.
Maybe most of these errors comes from incorrect installation.
First TRY to import Sample Rdc's from addon website, if it works (cape town specially) then try with your models generated with Rendercap.
Hope it works for you.
